Dockerfile

```diff
@@ -32,13 +32,12 @@ ARG JELLYFIN_VERSION=4.4.1-4
 
 # COPY qemu-arm-static /usr/bin/qemu-arm-static
 
-COPY setup_jellyfin_repo.sh setup_jellyfin_repo.sh
 
-RUN ./setup_jellyfin_repo.sh
-# No need for updating because the shell script above does that for us.
-# RUN apt update
-
-RUN apt install -y jellyfin-ffmpeg=$JELLYFIN_VERSION-$(awk -F'=' '/^VERSION_CODENAME=/{ print $NF }' /etc/os-release)
+# RUN apt update and install wget
+RUN apt -y update && apt -y install wget
+RUN wget https://repo.jellyfin.org/archive/ffmpeg/debian/4.4.1-4/jellyfin-ffmpeg_4.4.1-4-buster_amd64.deb
+RUN apt -y install ./jellyfin-ffmpeg_4.4.1-4-buster_amd64.deb
+RUN rm jellyfin-ffmpeg_4.4.1-4-buster_amd64.deb
```

Comment:
this results apt cache, wget and the downloaded .deb in final image layer. should use multi stage build to avoid this. or run all commands and cleanup in single run command.